index.html

```diff
@@ -30,6 +29,176 @@ <h1 class="name">Javed Shahbaz</h1>
     <section class="Projects">
         <h2 class="Ptitle">Projects</h2>
 
-    </section> 
+        <div class="Gallery">
+    </section>
```

Comment:
- [ ] You have colsed `<section>` tag with an unclosed `<div>` tag. Kindly remove `</section>` from here. You have already closed `<section>` tag in `line number 162` of your code, So you don't need to use `</section>` tag here. It is semantically incorrect. Kindly fix this issue.

![image](https://user-images.githubusercontent.com/60083437/219809463-cfca6622-15ed-4e81-baca-607d66bc06a0.png)